scripts/servers_test.py

```diff
@@ -1035,3 +1035,24 @@ def test_managed_webdriverio_with_explicit_args(self) -> None:
         self.assertIn('--params.devMode=False', program_args)
         # From suite='full'.
         self.assertIn('--suite abc', program_args)
+
+    def test_managed_acceptance_test_server_with_explicit_args(self) -> None:
+        popen_calls = self.exit_stack.enter_context(self.swap_popen())
+
+        puppeteer_acceptance_tests_dir_path = os.path.join(
+            common.CURR_DIR, 'core', 'tests', 'puppeteer-acceptance-tests')
+        spec_dir_path = os.path.join(
+            puppeteer_acceptance_tests_dir_path, 'spec')
+        test_file_path = os.path.join(spec_dir_path, 'test')
+
+        self.exit_stack.enter_context(servers.managed_acceptance_tests_server(
+            suite_name='test',  # pylint: disable=line-too-long
```

Comment:
Is the ignore needed? The line doesn't look long

Reply:
yeah removed. there is no need for that